clinica/pipelines/__init__.py

```diff
@@ -1 +1,31 @@
-# from .engine import Pipeline, postset
+from ..pydra.t1_linear import t1_linear_cli as pydra_t1_linear_cli  # noqa
+from ..pydra.t1_volume.create_dartel import cli as pydra_t1vol_cd_cli  # noqa
+from ..pydra.t1_volume.tissue_segmentation import cli as pydra_t1vol_ts_cli  # noqa
+from . import cli
+from .deeplearning_prepare_data import deeplearning_prepare_data_cli
+from .dwi_connectome import dwi_connectome_cli
+from .dwi_dti import dwi_dti_cli
+from .dwi_preprocessing_using_fmap import dwi_preprocessing_using_phasediff_fmap_cli
+from .dwi_preprocessing_using_t1 import dwi_preprocessing_using_t1_cli
+from .machine_learning import classification_cli
+from .machine_learning_spatial_svm import spatial_svm_cli
+from .pet_linear import pet_linear_cli
+from .pet_surface import pet_surface_cli, pet_surface_longitudinal_cli
+from .pet_volume import pet_volume_cli
+from .statistics_surface import statistics_surface_cli
+from .statistics_volume import statistics_volume_cli
+from .statistics_volume_correction import statistics_volume_correction_cli
+from .t1_freesurfer import t1_freesurfer_cli
+from .t1_freesurfer_longitudinal import (
+    t1_freesurfer_longitudinal_cli,
+    t1_freesurfer_longitudinal_correction_cli,
+    t1_freesurfer_template_cli,
+)
+from .t1_linear import flair_linear_cli, t1_linear_cli
+from .t1_volume import t1_volume_cli
+from .t1_volume_create_dartel import t1_volume_create_dartel_cli
+from .t1_volume_dartel2mni import t1_volume_dartel2mni_cli
+from .t1_volume_existing_template import t1_volume_existing_template_cli
+from .t1_volume_parcellation import t1_volume_parcellation_cli
+from .t1_volume_register_dartel import t1_volume_register_dartel_cli
+from .t1_volume_tissue_segmentation import t1_volume_tissue_segmentation_cli
```

Comment:
Looks to me like we are exposing a lot of symbols for nothing. In my opinion, it should be possible to do:

`clinica/pipelines/__init__.py`
```python
from . import (
    deeplearning_prepare_data,
    dwi_connectome,
    [...],
    t1_volume_register_dartel,
    t1_volume_tissue_segmentation,
)
```

and have individual submodules be responsible for importing their own CLI package(s) like:

`clinica/pipelines/t1_linear/__init__.py`
```python
import flair_linear_cli
import t1_linear_cli
```

which would actually run the CLI registration code.

Reply:
Good suggestion !
I re-organized the imports in [ffc5ff0](https://github.com/aramis-lab/clinica/pull/787/commits/ffc5ff0b16c2b5a1a2d77036eaa83deb1202cd1a)